content/doc/book/managing/script-approval.adoc

```diff
@@ -103,6 +103,13 @@ usages of in-process scripting. When the <<groovy-sandbox>> is disabled, or a
 method outside of the built-in list is invoked, the Script Security plugin will
 check the Administrator-managed list of approved scripts and methods.
 
+
+[IMPORTANT]
+====
+A script is approved for the use anywhere in Jenkins and is not tied to a specific job or any other specific use of the script.
```

Comment:
```suggestion
A script is approved for the use in any Jenkins feature and plugin that integrates with script approval and is not tied to a specific job or any other specific use of the script.
```

An attempt to be more explicit about this not just being a Pipeline thing. While this is mentioned further up on the page, better to phrase it to remind folks about it.